1907-05-20.xml

```diff
@@ -2296,10 +2320,11 @@
                     <p>An English Daily Newspaper, Established in 1880.</p>
                     <p>Editor &amp; Manager: R. Snelling.</p>
                     <p>Price: One Piastre Tariff.</p>
-                    <p><date when="1905-07-13">MONDAY, MAY 20, 1907</date>.</p>
+                    <p><date when="1907-05-20">MONDAY, MAY 20, 1907</date>.</p>
                 </div>
                 <div type="advert" xml:id="deg-ad-ecs01">
```

Comment:
this should be `<div type="item">`--no id needed (it's not an ad or boilerplate).